test/functorch/test_aotdispatch.py

```diff
@@ -4892,6 +4898,71 @@ def test_autocast(self):
             res = aot_mod(x)
         res.sum().backward()
 
+    @unittest.skipUnless(has_triton(), "Triton is unavailable")
+    def test_partition_cross_entropy(self):
+        """
+        In transformer, it usually hurts perf if the partitioner saves softmax
+        output for the backward pass. We can skip writing the softmax output
+        but recompute it based on softmax input and max/sum statistics.
+        """
+        B = 32
+        T = 1024
+        C = 768
+        V = 50257
+
+        linear = nn.Linear(C, V, bias=False, dtype=torch.bfloat16).cuda()
```

Comment:
Hi, may I suggest we replace `"cuda"` with the GPU_TYPE  in this case?  XPU also meet the  `has_triton` requirement.  You may also skip if not torch.cuda.is_available() for this case. Thanks.